source/_components/binary_sensor.august.markdown

```diff
@@ -2,14 +2,14 @@
 layout: page
 title: "August Binary Sensor"
 description: "Instructions on how to integrate your August devices into Home Assistant."
-date: 2018-02-17 22:00
+date: 2018-10-03 15:00
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: august.png
-ha_category: Doorbell
-ha_release: "0.64"
+ha_category: Doorbell, Door
+ha_release: "0.80"
```

Comment:
Please do not update the `ha_release` in this case, it is used to indicate the first version where the component was introduced to Home Assistant. This is just an addition to an existing component.